eng/Versions.props

```diff
@@ -30,6 +30,10 @@
     <!-- Dependency from https://github.com/dotnet/test-templates -->
     <MicrosoftDotNetTestProjectTemplates31PackageVersion>1.0.2-beta4.19465.1</MicrosoftDotNetTestProjectTemplates31PackageVersion>
   </PropertyGroup>
+  <PropertyGroup>
+    <!-- Dependency from https://github.com/dotnet/wcf -->
+    <SystemServiceModelPrimitivesPackageVersion>4.7.0-preview1.final</SystemServiceModelPrimitivesPackageVersion>
```

Comment:
We don't need this variable if we're not going to use it.

Reply:
My understanding is that it will be getting used as part of the process for pulling together the final set of packages for final release. Even though WCF doesn't get built together with the rest of the shared framework we do want our packages to ship togther with the rest of the .NET Core packages.

Reply:
This is adding a variable that is reference nowhere in the code. I don't think that is necessary.

Reply:
Correct, the MSBuild prop isn't necessary, only the Version.Details.xml so that the drop gathering tool can find WCF based on the .NET Core build.

Reply:
Removed.